pkgs/by-name/yt/ytmdesktop/package.nix

```diff
@@ -0,0 +1,97 @@
+{
+  lib,
+  asar,
+  binutils,
+  commandLineArgs ? "",
+  electron_30,
+  fetchurl,
+  makeDesktopItem,
+  makeWrapper,
+  nix-update-script,
+  stdenvNoCC,
+  zstd,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "ytmdesktop";
+  version = "2.0.5";
+
+  meta = {
+    changelog = "${finalAttrs.meta.downloadPage}/tag/v${finalAttrs.version}";
+    description = "A Desktop App for YouTube Music";
+    downloadPage = "https://github.com/ytmdesktop/ytmdesktop/releases";
+    homepage = "https://ytmdesktop.app/";
+    license = lib.licenses.gpl3Only;
+    mainProgram = finalAttrs.pname;
+    maintainers = [ lib.maintainers.cjshearer ];
+    inherit (electron_30.meta) platforms;
+    # While the files we extract from the .deb are cross-platform (javascript), the installation
+    # process for darwin is different, and I don't have a test device. PRs are welcome if you can
+    # add the correct installation steps. I would suggest looking at the following:
+    # https://www.electronjs.org/docs/latest/tutorial/application-distribution#manual-packaging
+    # https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/jitsi-meet-electron/default.nix
+    badPlatforms = lib.platforms.darwin;
+  };
+
+  passthru.updateScript = nix-update-script { };
+
+  desktopItem = makeDesktopItem {
+    desktopName = "Youtube Music Desktop App";
+    exec = finalAttrs.pname;
+    icon = finalAttrs.pname;
+    name = finalAttrs.pname;
+    genericName = finalAttrs.meta.description;
+    mimeTypes = [ "x-scheme-handler/ytmd" ];
+    categories = [
+      "AudioVideo"
+      "Audio"
+    ];
+    startupNotify = true;
+  };
+
+  nativeBuildInputs = [
+    asar
+    binutils
```

Comment:
Isn't that in stdenv?

Reply:
Yeah, but it must not be in `stdenvNoCC`, since the `ar` command isn't otherwise found.

Reply:
Then we probably shouldn't be using stdenvNoCC here, if we need part of cc 

```suggestion
```